.cloudgov/manifest.yml

```diff
@@ -10,6 +10,7 @@ applications:
         name: owasp-zap
     services:
       - pages-((env))-encryption
+      - headers
```

Comment:
Since we are creating a custom `user-agent` name the service name should be related to `user-agent`.